scripts/konflux-build-triage/process_pull_requests.py

```
@@ -0,0 +1,213 @@
+
+import json
+import re
+import subprocess
+import sys
+from html.parser import HTMLParser
+
+REPO = "openshift/sandboxed-containers-operator"
+
+
+def gh(*args):
+    result = subprocess.run(["gh"] + list(args), capture_output=True, text=True)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)